src/screens/DomainScreen/elements/Header/elements/ActionButtonGroup.js

```diff
@@ -137,14 +137,14 @@ const styles = StyleSheet.create({
     fontWeight: 'bold',
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(24),
-    color: COLORS.white
+    color: COLORS.white2
   },
   textButtonFollowing: {
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(24),
-    color: COLORS.anon_primary
+    color: COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in text color from `COLORS.white` to `COLORS.white2` and `COLORS.anon_primary` to `COLORS.signed_primary` are also acceptable, provided these new colors are defined and they fit into your new color scheme. 

```diff
-    color: COLORS.white
+    color: COLORS.white2

-    color: COLORS.anon_primary
+    color: COLORS.signed_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->